tests/runner_spec.lua

```diff
@@ -178,7 +194,15 @@ describe("runner", function()
       local config = assert(conform.get_formatter_config("test"))
       local ctx = runner.build_context(0, config)
       local cmd = runner.build_cmd("", ctx, config)
-      assert.equal("echo " .. vim.fs.dirname(vim.api.nvim_buf_get_name(bufnr)) .. " | patch", cmd)
+      assert.are.same(
+        shell(
+          vim.fn.exepath("echo")
+            .. " "
+            .. vim.fs.dirname(vim.api.nvim_buf_get_name(bufnr))
+            .. " | patch"
+        ),
+        cmd
+      )
```

Comment:
ehhh...it's getting annoying to maintain these tests. They're pretty low-value, so I think we can just delete them. I'll do that and add a new test for a shell-command formatter.